lib/tasks/reporting.rake

```diff
@@ -110,34 +109,33 @@ namespace :reporting do
         puts "    Updated #{updated_srun_count} SequenceRuns"
         puts "    Updated #{updated_run_count} Runs"
       end
-      clazz_id, class_hash, learner_id, learner_key = import_line.strip.split(",")
+      clazz_id, class_hash, user_id, learner_key, resource_link_id = import_line.strip.split(",")
```

Comment:
This was refactored by Noah, so we should merge in master and resolve merge conflicts. The new code uses the learner_id when the learner_key is missing.

Reply:
I think I just fixed this. I'll wait for it to tests to run now.